modules/core/shared/src/main/scala/lucuma/core/enum/GnirsPrism.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core
+package enum
+
+import cats.instances.string._
+import cats.syntax.eq._
+import lucuma.core.util.Enumerated
+
+/**
+ * Enumerated type for GNIRS Prism Turret.
+ * @group Enumerations (Generated)
```

Comment:
Are these still generated?  I thought we abandoned that idea.

Reply:
Not generated anymore. I think we agreed that the Scala enum is truth and database apps need to do a some kind of startup check to ensure consistency (not implemented).